### PR TITLE
Ankit | Test | Provide support of multiple deposit in single voucher in create/update flow (PAID)[BUG]

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cvwurra

### CODE REVIEW

File: apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.html [hunk 1]
```diff
@@ -42,7 +41,8 @@
                 </reactive-dropdown-field>
                 <div class="d-flex align-items-center w-50">
                     <input-field
-                        [name]="'amount_' + depositIndex"
+                        [id]="'amount_'+depositIndex"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.html]: format code

File: apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts [hunk 2]
```diff
@@ -139,8 +138,8 @@ export class PaymentDialogComponent implements OnInit, OnDestroy {
      */
     private getDepositFormGroup(): FormGroup {
         return this.formBuilder.group({
-            amount: ['', Validators.required],
-            accountUniqueName: ['', Validators.required],
+            amount: [''],
+            accountUniqueName: [''],
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts]:
```suggestion
            accountUniqueName: ['']
```

File: apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts [hunk 3]
```diff
@@ -204,7 +202,11 @@ export class PaymentDialogComponent implements OnInit, OnDestroy {
                     }
                 } else {
                     this.isBankSelected = false;
-                    this.paymentForm.get('accountUniqueName')?.patchValue('');
+                    let deposits = this.paymentForm.get('deposits') as FormArray;
+                    this.paymentForm.get('deposits')['controls']?.forEach((control: any, index: number) => {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts]: move into common method

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.html (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.html]: Done

File: apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts]: Done

File: apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/payment-dialog/payment-dialog.component.ts]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**